sal_interfaces/LinearStage/LinearStage_Commands.xml

```diff
@@ -66,16 +66,16 @@
 		<Explanation>http://sal.lsst.org</Explanation>
 		<item>
 			<EFDB_Name>homed</EFDB_Name>
```

Comment:
called `value`, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."